llama_stack/providers/remote/tool_runtime/tavily_search/config.py

```diff
@@ -18,10 +18,20 @@ class TavilySearchToolConfig(BaseModel):
         default=3,
         description="The maximum number of results to return",
     )
+    timeout: float = Field(
+        default=30.0,
+        description="HTTP request timeout in seconds",
+    )
+    connect_timeout: float = Field(
+        default=10.0,
+        description="HTTP connection timeout in seconds",
```

Comment:
I'm wondering if it would be enough to just set the request timeout, was the connect timeout required for it to work?

Reply:
What I mean is that I'm wondering if this would be enough?
> httpx.Timeout(timeout=self.config.timeout)

which should set the default timeout 

or maybe we could leave the write, connect and pool timeouts at their default (5) and only change the read timeout (I'm assuming its the read timeout you hit? )
> httpx.Timeout(timeout=5, read=self.config.timeout)

Essentially I'm trying to minimise the number of config options unless they are necessary.